csvapi/tableview.py

```diff
@@ -78,6 +78,12 @@ def add_filters_to_sql(self, sql, filters):
             elif comparator == 'contains':
                 wheres.append(f"[{column}] LIKE :filter_value_{normalized_column}")
                 params[f'filter_value_{normalized_column}'] = f'%{f_value}%'
+            elif comparator == 'less':
+                wheres.append(f"[{column}] <= :filter_value_l_{normalized_column}")
+                params[f'filter_value_l_{normalized_column}'] = float(f_value)
```

Comment:
This will fail if `f_value` can't be converted to a float. This is not necessarily a problem, depending on the way the exception is handled. If it's catched and returned through the API I think it's OK (like `{"error":"Can't parse float or smtg","ok":false}`). We should add a test for this.

Reply:
It does return the error properly: `{"error":"could not convert string to float: 'text'","error_id":null,"ok":false}`.
Agreed for the test :)